src/packs/tables/tables_Complications_jjtM55JUzmW6TaG2.json

```
@@ -0,0 +1,2554 @@
+{
+  "name": "Complications",
+  "description": "<p>A random table created from the contents of the Complications Folder.</p>",
```

We should mirror the book's language.
```suggestion
  "description": "<p>To maximize the sense that a complication is an unexpected part of your life, roll on this table.</p>",
```